scipy/special/tests/test_logsumexp.py

```diff
@@ -139,6 +139,11 @@ def test_logsumexp_b_shape():
 
     logsumexp(a, b=b)
 
+def test_logsumexp_precision():
+    a = [0, -40]
+    expected = 4.248354255291589e-18
+    assert_almost_equal(logsumexp(a), expected)
```

Comment:
```suggestion
    assert_allclose(logsumexp(a), expected, rtol=1e-18)
```
`assert_almost_equal` doesn't throw an error for precision this low, which might cause a false negative.